.build/.k.rev

```diff
@@ -1 +1 @@
-b65568ade007d54dd21667f4f56f2d7471bb6596
+4a9364d1b683d31cd8029cffc9217c7b230b657c
```

Comment:
i think we need to hold this K change until we all agree with the variable abstraction.

Reply:
New K is actually necessary as I explained below. The merge can wait.